<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommonAtomPropertiesWidget</class>
 <widget class="QWidget" name="CommonAtomPropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="charge_lbl">
     <property name="toolTip">
      <string>Formal charge</string>
     </property>
     <property name="text">
      <string>Charge:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="enhanced_stereo_lbl">
     <property name="toolTip">
      <string>Chiral center label for enhanced stereochemistry - ABS (absolute) is the default; AND / OR require a group identifier</string>
     </property>
     <property name="text">
      <string>Enhanced Stereo:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="isotope_lbl">
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Isotope number</string>
     </property>
     <property name="text">
      <string>Isotope:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="unpaired_electrons_lbl">
     <property name="toolTip">
      <string>Number of unpaired electrons - defines the radical character of the system</string>
     </property>
     <property name="text">
      <string>Unpaired Electrons:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="enhanced_stereo_combo">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="enhanced_stereo_sb">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Group identifier - must be an integer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QSpinBox" name="unpaired_elec_sb">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QSpinBox" name="charge_sb">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>-8</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="isotope_le">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
